product_docs/docs/pgd/5/raft/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+
+title: Proxies, Raft and Raft subgroups
+
+---
+
+PGD manages its metadata using a  Raft model where a top-level group spans all the data nodes in the PGD cluster. A Raft leader is elected by the top-level group and propagates the state of the top-level group to all the other nodes in the group. 
+
+!!! Hint What is Raft?
+Raft is an industry accepted algorithm for making decisions though achieving “consensus” from a group of separate nodes within a distributed system. 
+!!!
+
+It is essential for certain operations in the cluster that the Raft leader – such as adding and removing nodes and allocating ranges for [galloc](sequences/#pdg-global-sequences) sequences – is both established and maintained. 
+
+It also means that one half of the nodes in the top-level group, plus one additional node must be able to reach each other. So, in a top-level group with five nodes, at least three of the nodes must be reachable by each other to establish a Raft leader.
+
+## Raft and proxy routing
+
+One function that also uses Raft is proxy routing. Proxy routing requires that the proxies are able to coordinate writing to a data node within their group of nodes. This data node is the write leader. If the write leader goes offline, the proxies need to be able to switch to a new write leader, selected by the data nodes, to maintain continuity for connected applications.
+
+Proxy routing can be configured on a per node group basis in PGD 5, but the recommended configurations are "global" and "local" routing.
```

Comment:
For `bdr.commit_scope`, we do use `global` and `local`. But please note that while `global` matches and means all nodes, `local` there means just the single one PGD data node connected to.

I propose to use the term `regional` here.

Reply:
Those are the terms exposed in TPA in --enable-proxy-routing [global|local] 

Ideally, toplevel and subgroup would be the routing settings.

Reply:
let's keep this in sync with TPA